aten/src/ATen/native/cudnn/RNN.cpp

```diff
@@ -867,7 +867,7 @@ copy_weights_to_flat_buf_views(
   // Why do we pad to 5 dims here (and elsewhere)?
   // https://docs.nvidia.com/deeplearning/sdk/cudnn-api/index.html#cudnnRNNForwardTraining
   // expects descriptors padded to 3 dimensions.
-  x_desc.set(flat_buf_datatype, x_geom.sizes(), x_geom.strides(), 5);
+  x_desc.set(flat_buf_datatype, c10::asIntArrayRefSlow(x_geom.sizes()), c10::asIntArrayRefSlow(x_geom.strides()), 5);
```

Comment:
Since geometry is used by proper kernels, I wonder if it makes sense to follow the same naming convention as TensorImpl and have separate sizes and sym_sizes, so you don't have to do this. That will also give you a logical place to store "yes this is guaranteed to be non-symbolic" so you don't have to pessimize the code here

Reply:
I'm going to do this